lib/router/index.js

```diff
@@ -59,6 +60,8 @@ function throwIfNoRouter () {
 // Export the SingletonRouter and this is the public API.
 export default SingletonRouter
 
+export const withRouter = _withRouter
```

Comment:
Since we don't need to access `_withRouter` we can directly reexport the module here:

```javascript
export { default as withRouter } from './with-router'
```